operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -539,4 +546,56 @@ public static String fromAsciiBytes(byte[] bytes) {
         return new String(bytes, StandardCharsets.US_ASCII);
     }
 
+    /**
+     * Build basic HTTP authentication header value.
+     * 
+     * @param username Username.
+     * @param password Password.
+     * 
+     * @return Header value.
+     */
+    public static String buildBasicAuthValue(String username, String password) {
+        String credentials = join(":", username, password);
+        return format("Basic %s", Util.encodeToBase64(credentials));
+    }
```

Comment:
This seems to be used only by TO. So it should probably live there.

Reply:
Ok, moved to TO util. Thanks.